app/assets/javascripts/discourse/app/components/bootstrap-mode-notice.js

```diff
@@ -22,4 +24,16 @@ export default class BootstrapModeNotice extends Component {
       `/t/-/${this.siteSettings.admin_quick_start_topic_id}`
     );
   }
+
+  @computed("siteSettings.admin_quick_start_topic_id")
+  get adminGuideUrl() {
+    return getURL(`/t/-/${this.siteSettings.admin_quick_start_topic_id}`);
+  }
+
+  @computed("adminGuideUrl")
+  get userTipContent() {
+    return I18n.t("user_tips.admin_guide.content", {
+      admin_guide_url: this.adminGuideUrl,
+    });
+  }
```

Comment:
No need for `computed`, it's a glimmer component now 😃

```suggestion
  get adminGuideUrl() {
    return getURL(`/t/-/${this.siteSettings.admin_quick_start_topic_id}`);
  }

  get userTipContent() {
    return I18n.t("user_tips.admin_guide.content", {
      admin_guide_url: this.adminGuideUrl,
    });
  }
```